src/model/quotes.js

```diff
@@ -1071,7 +1071,7 @@ class QuotesModel {
   generateRequestHeaders (headers, noAccept) {
     const ret = {
       'Content-Type': 'application/vnd.interoperability.quotes+json;version=1.0',
-      Date: headers.date,
+      Date: (headers.date && (typeof headers.date !== 'string')) ? headers.date.toUTCString() : headers.date,
```

Comment:
I don't think I saw a test for this, but I think you said you'd worry about tests and coverage later?

Reply:
yup
there is a negative test, but will do the positive with the missing route tests

Reply:
Have added the extra tests for this